pkg/system/phase2_creating.go

```diff
@@ -1242,7 +1242,7 @@ func (r *Reconciler) ReconcileODFPersistentLoggingPVC(
 	log := r.Logger.WithField("func", "ReconcileODFPersistentLoggingPVC")
 
 	// Return if persistent logging PVC already exists
-	if pvc != nil {
+	if pvcName != nil {
```

Comment:
Just a code reading comment.

Somehow it doesn't add up.
if pvcName == nil the we will not enter this if and will not return.
Then  on line number 1253, we will print the name *pvcName.

Next just by checking *pvcName we are saying PersistentLoggingPVC exist and assigning ovcName to pvc.Name. What if pvcName is already have some name because it exist.

I think check on 151 should be the first and then we should check pvcName and act accordingly.

Reply:
Good catch, thanks.
I've fix to the correct field in 1253.